src/main/webapp/app/exercises/programming/shared/code-editor/file-browser/code-editor-file-browser.scss

```diff
@@ -77,6 +77,8 @@ File browser tree
             }
             button {
                 padding: 0;
+                border: none;
+                pointer-events: auto;
```

Comment:
Why is this necessary? When I remove this in the DevTools, everything seems to still be fine 🤔

Reply:
When the button is disabled, there was a weird border that I did not like ^^"

Also, it did not show the tooltip on hover, that tells you why the button is disabled. _It's weird how we had a tooltip explaining why a button is disabled that was not being displayed when the button was disabled_ O.o 
With ```pointer-events: auto``` the tooltip can now be displayed, even if the rename button is disabled